website/content/docs/platform/k8s/helm/openshift.mdx

```diff
@@ -23,7 +32,7 @@ on OpenShift:
 - Vault Helm v0.6.0+
 - Vault K8s v0.4.0+
 
-~> **Note:** Consul on OpenShift is supported since [Consul 1.9] (https://www.hashicorp.com/blog/introducing-openshift-support-for-consul-on-kubernetes). However, for highly available
+~> **Note:** Consul on OpenShift is supported since [Consul 1.9](https://www.hashicorp.com/blog/introducing-openshift-support-for-consul-on-kubernetes). However, for highly available
```

Comment:
```suggestion
~> **Note:** Support for Consul on OpenShift is available since [Consul 1.9](https://www.hashicorp.com/blog/introducing-openshift-support-for-consul-on-kubernetes). However, for highly available
```